tests/js_test/js-test-sn-collapse-button.cy.js

```diff
@@ -0,0 +1,86 @@
+describe('Test Sphinx Needs Collapse', () => {
```

Comment:
is `js-test-...` as prefix for the js-test file needed?
The folder is already called like this, so in my understanding we could shorten the file name a little bit. Just to `sn-collapse-button.cy.js`?